blogposts/2022/sourcegraph-extension-for-vs-code.md

```diff
@@ -0,0 +1,53 @@
+---
+title: 'How I use the Sourcegraph extension for VS Code'
+description: 'Sourcegraph’s extension for Visual Studio Code allows you to search millions of open source repositories without cloning them to your local machine or leaving your IDE. Here’s how one engineer integrates it into his workflow.'
+author: Murat Sutunc
+publishDate: 2022-02-08T00:00-07:00
+tags: [blog]
+slug: ways-to-use-sourcegraph-extension-for-vs-code
+heroImage: https://sourcegraphstatic.com/blog/vs-code-extension/sourcegraph-vs-code-extension.png
+socialImage: https://sourcegraphstatic.com/blog/vs-code-extension/sourcegraph-vs-code-extension.png
+published: true
+---
+
+![Explore millions of open source repositories right from your IDE](https://sourcegraphstatic.com/blog/vs-code-extension/sourcegraph-vs-code-extension.png)
+
+My name is [Murat Sutunc](https://handbook.sourcegraph.com/team#murat-sutunc), and I’m the Engineering Manager for the Growth and Integrations team at Sourcegraph. Our team currently owns the initial user journey, growth-oriented projects, our code host and third-party integrations (including our IDE and browser extensions), and Sourcegraph extensions. As someone who does a lot of code reviews and occasionally jumps in to fix small issues, I wanted to share with everyone how I use my favorite IDE (VS Code!) and Sourcegraph together.
+
+TL;DR 👉 The Sourcegraph VS Code extension is a really powerful tool that lets you search your code and 2M+ open source repositories, all from your IDE! It’s available on [the marketplace](https://marketplace.visualstudio.com/items?itemName=sourcegraph.sourcegraph)—no account needed, give it a try!
```

Comment:
Wondering if we should give this some kind of different style treatment, e.g. like at the top of this post: https://about.sourcegraph.com/blog/log4j-log4shell-0-day/?

@muratsu @iskyOS

Reply:
@rebeccadee I like calling out that section! We could definitely use that alert box style.